lib/User/Manager.php

```diff
@@ -136,6 +136,22 @@ private function checkAccess() {
 	public function getConnection() {
 		return $this->access->getConnection();
 	}
+
+	/**
+	 * Get a list with the configured exposed attributes.
+	 * The `getAttributes` method will contain these exposed attributes, and all
```

Comment:
Not sure exactly what the comment means. Maybe
```suggestion
	 * The `getExposedAttributes` method will return an array of these exposed attributes, and all
```

Reply:
The `getAttributes` method refers to the method below this one.
Basically, the `$manager->getExposedAttributes()` will return the attributes exposed by the admin, and the `getAttributes()` will return everything, including the exposed attributes. In other words, the `getAttributes` will contain the `getExposedAttributes`.

I guess I need to figure out how to explain it better.